cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/TargetGroupPortPair.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.cloud.model;
+
+import java.util.Objects;
+
+public class TargetGroupPortPair {
```

Comment:
**suggestion:** I know that we're not huge fans of comments around these parts, but I think it would be good to mention in a Java doc where we're getting the target group from, and that this has to do with Cloud load balancers in particular.